core/integration/monitor_tests.go

```diff
@@ -43,89 +42,91 @@ amFdON6OhjYnBmJWe4fVnbxny0PfpkvXtg==
 // TestMonitor verifies that the monitor correctly verifies transitions between epochs.
 func TestMonitor(ctx context.Context, env *Env, t *testing.T) {
 	// setup monitor:
-	resp, err := env.Cli.GetDomain(ctx, &pb.GetDomainRequest{DomainId: env.Domain.DomainId})
-	if err != nil {
-		t.Fatalf("Couldn't retrieve domain info: %v", err)
-	}
-	signer, err := pem.UnmarshalPrivateKey(monitorPrivKey, "")
+	privKey, err := pem.UnmarshalPrivateKey(monitorPrivKey, "")
 	if err != nil {
 		t.Fatalf("Couldn't create signer: %v", err)
 	}
-	mapVerifier, err := tclient.NewMapVerifierFromTree(resp.GetMap())
+	signer := crypto.NewSHA256Signer(privKey)
```

Comment:
move this down to the line preceding that declaring `store`

Reply:
done